### DISCUSSION TITLE
Generalizability Analysis Issues

### DISCUSSION THREAD

**Discussion by Papa-26** [G-String-Legacy/G_String#1] [Announcements]
See below

**Comment by wesleysucr** [comment]
Hello,
I'm having a really hard time setting up my data in an csv file for analysis in G String. I have 74 raters that rated 3 teachers using a measure that uses 4 items to generate a total score. I believe this gives me 6 facets with the total score serving as the facet of differentiation and rater, item 1, item 2, item 3, and item 4 as facets of generalization. All raters rated all teachers using all items (fully crossed). I am unsure how to arrange this data in columns and rows for analysis in G String. Any help would be appreciated.

**Comment by Papa-26** [comment]
Hi wesleysucr,
Let's get your problem solved: In fact, you only have three facets - T (teachers, facet of differentiation) R (raters, facet of generalization), and I (item, facet of generalization). All this assuming that you are interested in determining, how well your design can discriminate between different teachers. See page 20 of the guide. If this is your design, then you have 3 possible ways of setting up your data file:
a) 1 line per score with 3 index columns and one data column
b) 1 line per teacher and rater, with 2 index columns and 4 data columns
c) 1 line per teachet, with one index columns (optional) and 296 data columns (R x I)
I would think that option b0 is the most practical.
Two caveats: 
    i. make sure that all 4 items have the same metric, e.g. 1 ...5, or something like that.
   ii. Your items must be well chosen to reach statistical significance with a sample size of only 3 teachers, but it makes for a goo pilot study.

All the best,
Ralph
